src/platform/user/authentication/config/staging.config.js

```diff
@@ -85,4 +85,17 @@ export default {
     oAuthOptions: defaultMobileOAuthOptions,
     externalRedirectUrl: EXTERNAL_REDIRECTS[EXTERNAL_APPS.VA_OCC_MOBILE],
   },
+  [EXTERNAL_APPS.ARP]: {
+    allowedSignInProviders: { ...defaultSignInProviders }, // TODO: Confirm we want all default sign-in providers
```

Comment:
Might just be ID.me and Login.gov?

Reply:
@amprokop In other words, while [VA.gov](VA.gov) behavior is:
```
{
  allowedSignInProviders: {
    logingov: true,
    idme: true,
    dslogon: true,
    mhv: true,
  },
  allowedSignUpProviders: {
    logingov: true,
    idme: true,
  },
}
```
We will actually prefer to be more restrictive and exclude existing `dslogon` and `mhv` accounts from signing in?

Reply:
Hi @nihil2501 - yes, we want to exclude existing `dslogon` + `mhv` (this was confirmed in an enablement team meeting today)